tests/AeroSharp.IntegrationTests/DataAccess/KeyValue/KeyValueStoreTests.cs

```diff
@@ -432,6 +434,8 @@ public async Task When_writing_a_record_that_does_exist_update_is_successful()
                 WaitTimeInMilliseconds = 10,
                 WithExponentialBackoff = true
             };
+
+            await _recordOperator.DeleteAsync(UnoccupiedRecord, new WriteConfiguration(), default);
```

Comment:
Why is this delete needed here? The current set of integration tests are passing and it doesn't look like you're using `UnoccupiedRecord` below. 👀

Reply:
sorry this is a fat finger, removing